testkraken/workflowregtest.py

```diff
@@ -556,6 +556,16 @@ def _validate_envs(self):
                     raise SpecificationError(
                         "Every value in fixed_env element must be a dictionary or list."
                     )
+    def download_datalad_repo(self):
+        if not self.params["data"].get("url"):
+            raise ValueError(
+                "A value for url must be provided if the data "
+                "type is datalad_repo "
+                )
+        # Get directory name for repository
+        import pdb;pdb.set_trace()
+        dirname = Path()
```

Comment:
this is where I have gotten to. Next is integrating the functionality in data_management.py and deleting the superfluous stuff.